perllib/FixMyStreet/App/Form/Waste/Bulky/Shared.pm

```diff
@@ -302,6 +302,18 @@ sub validate {
         }
     }
 
+    if ( $self->current_page->name eq 'location' ) {
```

Comment:
I think this could be done as a `validate_method` on the `location` field? Just to keep it together; would have the same effect.

Reply:
In fact, if you move it to the `update_field_list` of the `location` page - you can see the special case things happening there already - I think all you need to do is set `$fields->{location}{maxlength}` and then the error handling is done for you, plus the client side will have the maxlength set so people can't go over anyway.

Reply:
If by client side you mean the JS picking up on the error before the form is submitted, that does not happen - is it meant to?

Reply:
Nothing JS involved, it sets `maxlength` on the HTML, which browsers prevent typing longer than in the box. I've just tried it locally, seems to work fine, maxlength=250 is present on the textarea and I can't enter more than 250 characters in the box.